docs/dev/table/sql/alter.md

```diff
@@ -134,4 +135,27 @@ Set one or more properties in the specified table. If a particular property is a
 ALTER DATABASE [catalog_name.]db_name SET (key1=val1, key2=val2, ...)
 {% endhighlight %}
 
-Set one or more properties in the specified database. If a particular property is already set in the database, override the old value with the new one.
\ No newline at end of file
+Set one or more properties in the specified database. If a particular property is already set in the database, override the old value with the new one.
+
+## ALTER FUNCTION
+
+{% highlight sql%}
+ALTER [TEMPORARY|TEMPORARY SYSTEM] FUNCTION 
+  [IF EXISTS] [catalog_name.][db_name.] function_name 
+  AS identifier [LANGUAGE JAVA|SCALA|
+{% endhighlight %}
+
+Alter a catalog function with the new identifier and optional language tag. If a function doesn't exist in the catalog, an exception is thrown.
+
```

Comment:
maybe also document that 'identifier' for java/scala is the class name?

Reply:
Good suggestion.